exercises/nucleotide-count/description.md

```diff
@@ -1,11 +1,19 @@
-Given a single stranded DNA string, compute how many times each nucleotide occurs in the string.
+Most people inherit not only their socioeconomic class from their parents, but also DNA (deoxyribonucleic acid), a set of chemical instructions that influence how their bodies are constructed!
```

Comment:
I'm not sure about the "Most people inherit not only their socioeconomic class from their parents" introduction, as I can imagine this could be off-putting to some people.

Reply:
It puts me off society, too ;)

I'll see if I can come up with a good alternative.

Reply:
Maybe remove the not only part and say we inherit heriditary aspects through DNA?